File: Programming Basics/C++/Introduction/FoodDelivery/FoodDelivery.cpp
#include <iostream>

int main()
{
    int chicken, fish, vegetarian;
    std::cin >> chicken >> fish >> vegetarian;

    double basePrice = chicken * 10.35 + fish * 12.40 + vegetarian * 8.15;
    double desertPrice = 0.2 * basePrice;

    double total = basePrice + desertPrice + 2.5;

    std::cout << total << std::endl;
}

File: Programming Basics/JavaScript/Introduction/Exercises/food-delivery.js
function solve(input) {
    const chicken = Number(input[0]);
    const fish = Number(input[1]);
    const vegetarian = Number(input[2]);

    const basePrice = chicken * 10.35 + fish * 12.4 + vegetarian * 8.15;
    const desertPrice = 0.2 * basePrice;

    const total = basePrice + desertPrice + 2.5;
    console.log(total);
}
